web_app/models/AC_model_soares1999.py

```diff
@@ -14,34 +14,29 @@ class Soares1999Model(CorrosionModel):
         Marine Structures, 12(6), 425-445 (1999). Elsevier.
     """
 
-    def __init__(self, parameters: Optional[Dict[str, float]] = None):
-        super().__init__(model_name='Reliability of Maintained, Corrosion Protected Plates')
-        self.steel = "Steel"
-        self.parameters = parameters if parameters else self._get_parameters()
+    def __init__(self, json_file_path: str):
+        super().__init__(json_file_path=json_file_path, model_name='Soares1999Model')
+        self.parameters: Dict[str, float] = {}
 
-    def _get_parameters(self) -> Dict[str, float]:
-        """Prompts the user to input values for all parameters and returns a dictionary of the parameters."""
+    def display_parameters(self) -> None:
+        """Prompts the user to input values for all parameters and stores them in the parameters dictionary."""
         limits = {
             'd_inf': {'desc': 'Long term thickness of corrosion wastage', 'lower': 0.001, 'upper': 1000, 'unit': 'mm'},
```

Comment:
Here as well please

Reply:
Maybe the function can be implemented in corrosion_model parent class and used in all child classes.

Reply:
Works as intended